saleor/order/models.py

```diff
@@ -82,6 +82,12 @@ def ready_to_confirm(self):
         """Return unconfirmed orders."""
         return self.filter(status=OrderStatus.UNCONFIRMED)
 
+    def overpaid(self):
+        """Return orders that are overpaid."""
+        # TODO: check what (captured_amount and not is_active) means
+        qs = self.annotate(amount_paid=Sum("payments__captured_amount", distinct=True))
```

Comment:
@fowczarek  What do you think? We are going to list overpaid orders on the dashboard home page which means that this query will be executed a lot.

Reply:
Apparently someone has already added a denormalized field in May 2021, I've used it instead and added db indexes for the expression